src/sources/http.rs

```diff
@@ -199,7 +199,7 @@ fn body_to_lines(buf: Bytes) -> impl Iterator<Item = Result<Bytes, ErrorMessage>
     })
 }
 
-fn decode_body(body: Bytes, enc: Encoding) -> Result<Vec<Event>, ErrorMessage> {
+pub fn decode_body(body: Bytes, enc: Encoding) -> Result<Vec<Event>, ErrorMessage> {
```

Comment:
I suggest limiting the visibility to `pub(super)` here.

Reply:
Following @jszwedko comment I relocated the shared code between the `http` &  `datadog` sources in `src/sources/util/http.rs` and adjusted the visibility accordingly.